front/app/containers/MainHeader/NotificationMenu/components/InternalCommentNotification/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { memo } from 'react';
+import { isNilOrError, stopPropagation } from 'utils/helperUtils';
+
+// services
+import { IInternalCommentNotificationData } from 'api/notifications/types';
+
+// i18n
+import messages from '../../messages';
+import { FormattedMessage, MessageDescriptor } from 'utils/cl-intl';
+
+// components
+import NotificationWrapper from '../NotificationWrapper';
+import Link from 'utils/cl-router/Link';
+import { DeletedUser } from '../Notification';
+
+interface Props {
+  notification: IInternalCommentNotificationData;
+}
+
+const mapPostTypeToLink = (
+  notification: IInternalCommentNotificationData
+): string => {
+  switch (notification.attributes.post_type) {
+    case 'Idea':
+      return `/admin/ideas/${notification.attributes.post_id}`;
```

Comment:
I think we wanted to link to the input manager in the project. So URL is /admin/projects/:projectId/ideas/:ideaId.

Reply:
(Need to check the specs or ask on Slack.)

Reply:
You're right, I remember us discussing input on project. Changing now